src/core/logical_layers/utils/getMutualExcludedActions.ts

```
@@ -17,24 +17,17 @@ export const getMutualExcludedActions = (() => {
   let enabledLayers: Set<string> | null = null;
   enabledLayersAtom.subscribe((s) => (enabledLayers = s));
 
-  let layersSettings: Map<string, AsyncState<LayerSettings | null>> | null =
-    null;
+  let layersSettings: Map<string, AsyncState<LayerSettings | null>> | null = null;
   layersSettingsAtom.subscribe((s) => (layersSettings = s));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)